packages/common/src/dto/payload.dto.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  PersonalInformationDTO,
+  ContactInformationDTO,
+  SkillInformationDTO,
+  AvailabilityDTO,
+} from '.';
+
+export class PayloadDTO {
```

Comment:
In my nested validator PR I've created a very similar class. I will combine the two after this merges.

Reply:
sorry, I knew this will conflict with yours but the submission was not working without this class